fix: correct path to internal types

crookedneighbor opened the pull request:
Was getting typescript compilation errors when using the module:

```
npm run typescript

> scryfall-client@0.23.1 typescript
> tsc --declaration

node_modules/@scryfall/api-types/src/objects/Card/values/ImageUris.ts:1:21 - error TS2307: Cannot find module 'src/internal' or its corresponding type declarations.

1 import { Uri } from "src/internal";
                      ~~~~~~~~~~~~~~

node_modules/@scryfall/api-types/src/objects/Card/values/PurchaseUris.ts:1:21 - error TS2307: Cannot find module 'src/internal' or its corresponding type declarations.

1 import { Uri } from "src/internal";
                      ~~~~~~~~~~~~~~

node_modules/@scryfall/api-types/src/objects/Card/values/RelatedUris.ts:1:21 - error TS2307: Cannot find module 'src/internal' or its corresponding type declarations.

1 import { Uri } from "src/internal";
                      ~~~~~~~~~~~~~~


Found 3 errors in 3 files.

Errors  Files
     1  node_modules/@scryfall/api-types/src/objects/Card/values/ImageUris.ts:1
     1  node_modules/@scryfall/api-types/src/objects/Card/values/PurchaseUris.ts:1
     1  node_modules/@scryfall/api-types/src/objects/Card/values/RelatedUris.ts:1
```

Correcting these paths to point to the correct relative path fixed the issue for me.

crookedneighbor committed:
fix: correct path to internal types

scarletcs commented on src/objects/Card/values/ImageUris.ts:
```diff
@@ -1,4 +1,4 @@
-import { Uri } from "src/internal";
+import { Uri } from "../../../../src/internal";
```
```suggestion
import { Uri } from "../../../internal";

```

scarletcs commented on src/objects/Card/values/PurchaseUris.ts:
```diff
@@ -1,4 +1,4 @@
-import { Uri } from "src/internal";
+import { Uri } from "../../../../src/internal";
```
```suggestion
import { Uri } from "../../../internal";

```

scarletcs commented on src/objects/Card/values/RelatedUris.ts:
```diff
@@ -1,4 +1,4 @@
-import { Uri } from "src/internal";
+import { Uri } from "../../../../src/internal";
```
```suggestion
import { Uri } from "../../../internal";

```

scarletcs committed:
rm redundant src from relative path

These paths backed out of src then back into src, but we can/should cancel those out.
